specification/metrics/semantic_conventions/instrumentation/kafka.md

```diff
@@ -36,6 +36,16 @@ This document defines how to apply semantic conventions when instrumenting Kafka
 | messaging.kafka.leader.elections             | Counter       | Int64      | elections | `{election}` | Leader election rate (increasing values indicates broker failures). | | |
 | messaging.kafka.leader.unclean-elections     | Counter       | Int64      | elections | `{election}` | Unclean leader election rate (increasing values indicates broker failures). | | |
 | messaging.kafka.brokers                      | UpDownCounter | Int64      | brokers   | `{broker}`   | Number of brokers in the cluster. | | |
+| messaging.kafka.brokers.count                | Gauge         | Int64      | brokers   | `{broker}`   | Number of brokers in the cluster. (Replacement for messaging.kafka.brokers) | | |
```

Comment:
Should we be consistent with the use of singular and plural metric names? We have mixed use here, for example `requests` plural, `topic` singular, and `partition` appears in singular and plural forms.